src/ReleaseHistory.md

```diff
@@ -14,56 +14,49 @@
 - UEE => eliminate unhandled exceptions in engine
 
 ## **v4.1.0** UNRELEASED
+* BUG: Fix BA2004 HASH not SHA256 message - was hardcoded to SHA-1. [#865](https://github.com/microsoft/binskim/pull/865)
+* DEP: Update Sarif.Sdk submodule from [120fae3 to 2d52c53](https://github.com/microsoft/sarif-sdk/compare/120fae3...2d52c53). Full [SARIF SDK Release History](https://github.com/microsoft/sarif-sdk/blob/2d52c53/src/ReleaseHistory.md).
+* BRK: Remove `--verbose` command-line option (in favor of `--level` and `--kind`). [#853](https://github.com/microsoft/binskim/pull/853)
+* BRK: Remove `--hashes` command-line option (in favor of `--insert Hashes`). [#853](https://github.com/microsoft/binskim/pull/853)
+* BUG: Fix `BA2022.SignSecurely` unhandled `InvalidOperationException`: `Unrecognized crypto HRESULT: 0x80096011`, which is `TRUST_E_MALFORMED_SIGNATURE`, by refreshing `CryptoError` enum with latest data from Windows SDK for Windows 11 (10.0.22621.0). [850](https://github.com/microsoft/binskim/pull/850)
 * BUG: Probe local symbols directory for PDBs in all code paths. [828](https://github.com/microsoft/binskim/pull/828)
 * BUG: Add missing output in PDB load tracing (enabled by `--trace PdbLoad`. [828](https://github.com/microsoft/binskim/pull/828)
 
 ## **v4.0.0**
-* BUG: Fix unhandled `ArgumentException` in `Enum.TryParse` on passing `PdbLoad` value to `--trace` command-line argument. [821](https://github.com/microsoft/binskim/pull/821)
```

Comment:
>unhandled [](http://example.com/codeflow?start=11&length=9)

Nice, well done, eliding the historical stuff as we close on the actual release. 

Also helpful point to add to our CORN/contributor's guidance. Also you've sorted out items, well done, so that they are grouped and listed in the appropriate order.

We do this, by the way, to frontload the most important breaks at the top of the list. the group itself helps resolve merge issues as well, if you simply insert or append, everyone gets conflicts with each other. details for the CORNy documentation.